app/components/agenda-manager/index.js

```diff
@@ -13,35 +10,20 @@ import {task} from "ember-concurrency-decorators";
 
 /** @extends {Component<Args>} */
 export default class AgendaManagerIndexComponent extends Component {
-  /** @type AgendaData */
-  @tracked popup = false;
-  @tracked editModalVisible = false;
   @tracked itemToEdit = null;
 
-  constructor(...args) {
-    super(...args);
+  get editModalVisible() {
+    return Boolean(this.itemToEdit);
```

Comment:
I slightly prefer the double negative (as stupid as that syntax is) cause of the `new Boolean()` trap
but Im more than ok with leaving it as is

Reply:
I'm unfamiliar with the `new Boolean()` trap, can you explain / link me something? :smile: 

I just go for this version because I think it's more readable, but I don't mind switching either if there are downsides to this.

Reply:
try this in your console:
```js
let b = new Boolean(false);
b ? "true" : "false";
```

Basically using the Boolean constructor can lead to confusing results. You're using it as a function here, which does what you expect. But coming from java or other languages seeing a function call with a PascalCase name can trigger a "huh I guess they forgot `new` here"

[more details](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)

Reply:
Ah yea. I see what you mean. Well, there are always code reviews to catch that, and an [eslint rule](https://eslint.org/docs/rules/no-new-wrappers) as well if things are _really_ getting out of hand :smile:.

I still prefer the `Boolean` version myself, since I think it's more readable. the `!!` version is also harder to google for. At least it was when I encountered it for the first time a couple of years ago :smile:.

Anyway, I think this is one of those things that should just be done consistently across a codebase so I'll change it.

Reply:
Changed to `!!`

Reply:
oh yea didnt know eslint would catch that
all good then

Reply:
It doesn't by default though. That rule isn't part of the recommended ruleset.